pkg/controller/controller_broker.go

```diff
@@ -371,16 +372,15 @@ func (c *controller) reconcileClusterServiceBroker(broker *v1beta1.ClusterServic
 				continue
 			}
 			glog.V(4).Infof(
-				"%s %q: ClusterServicePlan (K8S: %q ExternalName: %q) has been removed from broker's catalog; marking",
-				typeCSB, broker.Name, existingServicePlan.Name, existingServicePlan.Spec.ExternalName,
+				"%s %q: %s has been removed from broker's catalog; marking",
+				typeCSB, pretty.ClusterServicePlanName(existingServicePlan),
```

Comment:
looks like the broker name is dropped from this message

Reply:
fixed here: https://github.com/kubernetes-incubator/service-catalog/pull/1442/files#diff-823813a64f69ecf73ed0f982e7112da0R382